Relecture_sebastien_030225

sebastiensamyn opened the pull request:
Bonjour, 

Voici ma relecture du module Rappels. En gros: 
-j'ai simplifié la 1ere partie (R au ministère qui semblait plutôt dater de quand arrivait et qu'il fallait justifier son utilisation, ce qui n'est plus le cas)
-J'ai passé les rappels sur la manipulation de données avant R de base (plus simple de commencer par là) et regrouper tout ce ce qui concerne R de base ensemble
-Sur la manipulation de données, j'ai comparé avec R initiation et ajouter qq diapos (il n'y avait rien sur across() et j'ai ajouté en fin une liste d'autres fonctions utiles et vues en R initiation (voir s'il faut développer ou bien si on en reste là, ainsi qu'une diapo sur rio pour l'import/export)
- Sur R de base, j'ai mis les matrices à la fin (j'hésitais à supprimer, vu le peu d'usage qu'on en a en manipulation de données)

Je n'ai pas eu le temps de le faire mais je voulais aussi réutiliser pas mal d'illustrations de la formation R initiation. Je vais essayer aussi de proposer des exercices sur cette partie. Au niveau de la forme, il y a encore qq petits trucs (vu que je découvrais Quarto) mais pas trop grave (des images et nos de pages qui se chevauchent, des undefined en haut à droite qui se baladent). 

Dites-moi ce que vous en pensez. 

Bon début de semaine, 

Sébastien

sebastiensamyn committed:
test

sebastiensamyn committed:
Relecture_rappels_Sebastien_1

ddotta commented:
Hello @sebastiensamyn 
🙏 pour ta PR !

- Nickel pour la partie justification de l'utilisation de R
- Ok pour commencer par la manipulation de données avec dplyr avant de basculer sur R base
- Sur le across() on l'aborde dans le module sur les fonctions, donc on l'enleverait bien de cette partie rappels.

- Pour les autres ajouts et les exercices je te propose de refaire une autre pull request pour ne pas mélanger les 2 sujets.
- Pour les histoires de mise en forme, pas de soucis je regarde demain après avoir mergé :)

Encore merci pour ta contribution !

sebastiensamyn commented:
Hello Damien, 

Ok pour across(), pas la peine de faire doublon, j'aurais dû jeter un oeil avant sur la partie Fonctions (je n'avais plus en tête qu'on l'abordait à ce moment là).

Ok aussi pour la 2e pull request éventuelle sur les ajouts (je vais essayer mais pas sûr d'avoir le temps, il faut que je travaille un peu aussi pour le Srise Bretagne :-) ). 

Il me semble que tu m'avais parlé de conflits à régler, j'ai à intervenir ou bien tu as tout ce qu'il te faut? Si j'ai une action à faire, comment je dois procéder? 

Bonne journée, 

Sébastien

ddotta commented:
Ok Sébastien n'hésite pas à refaire une autre pull request pour la partie pratique 👍  
Pour les conflits, oui il y en a 2 (ils sont indiqués sur cette page), je pense que les fichiers quarto ont été modifiés après que tu aies créé ta branche, ce qui a généré le conflit.
Pas d'action de ta part, je m'en occupe aujourd'hui.
A+

ddotta committed:
fix: résolution conflit

Merge branch 'main' of https://github.com/SSM-Agriculture/formation-R-perf-01-rappels into _relecture_sebastien

# Conflicts:
#	02_objets.qmd
#	04_traitement-de-donnees.qmd

ddotta commented:
@sebastiensamyn 
J'ai mergé ta PR, c'est ok pour les conflits.
Encore merci pour ta contribution, cela a bien enrichi le diaporama.

Quelques remarques pour tes prochaines contributions : 
- Pour les chunks et éviter les problèmes d'évaluation dans le diaporama, nous avons pris l'habitude de les spécifier avec
````
```{.r}
```
````
- Pour les problèmes de mise en forme et notamment les retours à la ligne, il faut mettre des doubles espaces en fin de ligne lorsqu'on veut passer à la ligne.
- Globalement, l'éditeur visuel de RStudio rend quelques services mais perso je ne suis pas fan parce qu'il génère du bruit dans la syntaxe md générée et je préfère du coup passer par l'éditeur source directement.

A+

ddotta commented:
Pi @dicorynia et @hlemaintec
